### ISSUE TITLE
Install resource folder automatically if detected

### ISSUE THREAD

**Issue by ammaraziz** [Wytamma/snk#26]
Hi Wytamma,

When installing a pipeline that is following [snakemake best practices](https://snakemake.readthedocs.io/en/stable/snakefiles/deployment.html#distribution-and-reproducibility), there's usually a resource folder. Similar to detected `config/config.yaml` can `snk` detected and install `resources/` automatically (recursively) ?

<3 `snk`

**Comment by ammaraziz** [comment]
As an aside, does the `.snk` file contents when I manually referenced primers in `resources/` folder look okay?
```
resources:
- !!python/object/apply:pathlib.PosixPath
  - wfi_snk
  - resources
  - adapters
  - RSV_right.fa
- !!python/object/apply:pathlib.PosixPath
  - wfi_snk
  - resources
  - adapters
  - RSV_left.fa
```
I get this error:
```
yaml.constructor.ConstructorError: could not determine a constructor for the tag 'tag:yaml.org,2002:python/object/apply:pathlib.PosixPath'
```

**Comment by Wytamma** [comment]
Hey @ammaraziz can you provide an example for the resources folder? You can use `-r resources/` when you run the pipeline and that should copy the resources folder to the current directory at runtime. I will have to add an option to configure this properly.

As for the .Snk file this looks like an error! Will see what’s going wrong here… they should just  be specified as string paths in the .Snk file. looks like it’s trying to serialise a Path object.

**Comment by ammaraziz** [comment]
> You can use -r resources/ when you run the pipeline and that should copy the resources folder to the current directory at runtime.

This fixes it, nice! 

I think `-r resources/` should be the default when running the pipeline if `resources/` is present during installation. 

structure of the resource folder:
```
├── adapters
│   ├── FLUh1_tile_left.fa
│   ├── FLUh1_tile_right.fa
│   ├── FLUh3_tile_left.fa
│   ├── FLUh3_tile_right.fa
│   ├── FLU_left.fa
│   ├── FLU_right.fa
│   ├── RSV_left.fa
│   ├── RSV_right.fa
│   ├── RSV_tile_left.fa
│   └── RSV_tile_right.fa
└── colors
    ├── cov.tsv
    ├── flu.tsv
    └── rsv.tsv
```

**Comment by Wytamma** [comment]
Hey @ammaraziz I think I've fixed these issues in v0.6.7. Now the default behaviour is to copy the resources folder to the working directory at runtime.

**Comment by ammaraziz** [comment]
Very cool, tested and working! I did run into a small issue, if I rerun the pipeline (say if it stopped or I changed a setting), I get this error:

```
FileExistsError: Resource 'resources' already exists!
```

Looking at the code, it checks if the `resource` folder exists, but I wonder if this is needed? If I understand the snakemake best practices correctly, the resources folder contains data shipped during installation (eg references, primers) I don't think the resources folder changes between runs.
